packages/client-sdk-nodejs/src/index.ts

```diff
@@ -50,6 +50,7 @@ import * as CacheSortedSetIncrementScore from '@gomomento/sdk-core/dist/src/mess
 import * as CacheSortedSetRemoveElement from '@gomomento/sdk-core/dist/src/messages/responses/cache-sorted-set-remove-element';
 import * as CacheSortedSetRemoveElements from '@gomomento/sdk-core/dist/src/messages/responses/cache-sorted-set-remove-elements';
 import * as ItemGetType from '@gomomento/sdk-core/dist/src/messages/responses/item-get-type-api';
+import * as ItemGetTtl from '@gomomento/sdk-core/dist/src/messages/responses/item-get-ttl-api';
```

Comment:
Same thing here, wonder why only ItemGetType/TTL don't have that Cache Prefix like the other Apis?

Reply:
This is a really good question... if we could start over from scratch today I think we would not have that prefix at all, but since we can't, we should probably be consistent on this... @eaddingtonwhite can we change both of those? I can add a commit to your PR if you don't have time.

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/593/commits/809d41496a45acfbdc7da370454988da5ef1a861